api/Controllers/EventController.cs

```diff
@@ -15,19 +15,33 @@ public class EventController : ControllerBase
 	{
 		private readonly IEventService _eventService;
 		private readonly IUserService _userService;
+		private readonly IAttendanceService _attendanceService;
 		private readonly ICategoryService _categoryService;
 		private readonly IPictureService _pictureService;
 		private readonly IMapper _mapper;
 
-		public EventController(IEventService eventService, ICategoryService categoryService, 
-								IUserService userService, IPictureService pictureService, IMapper mapper){
+		public EventController(IEventService eventService, ICategoryService categoryService,
+								IUserService userService, IAttendanceService attendanceService,
+								IPictureService pictureService, IMapper mapper)
+		{
 			_eventService = eventService;
+			_attendanceService = attendanceService;
 			_categoryService = categoryService;
 			_userService = userService;
 			_pictureService = pictureService;
 			_mapper = mapper;
 		}
 
+		public async void AutoJoin(Event createdEvent)
```

Comment:
This should be a private method since it's used only internally.
Also, I've noticed that the last method - checkBackgroundImage, starts with a lowercase letter.
Dotnet methods should start with a Capital letter.